src/components/directorybrowser/directorybrowser.js

```diff
@@ -243,70 +243,70 @@ import alert from '../alert';
+            const fileOptions = {
+                includeDirectories: true
             };
-            this.close = () => {
-                if (currentDialog) {
-                    dialogHelper.close(currentDialog);
+            if (options.includeDirectories != null) {
+                fileOptions.includeDirectories = options.includeDirectories;
+            }
+            if (options.includeFiles != null) {
+                fileOptions.includeFiles = options.includeFiles;
+            }
+            Promise.all([getSystemInfo(), getDefaultPath(options)]).then(
+                responses => {
+                    const systemInfo = responses[0];
+                    const initialPath = responses[1];
+                    const dlg = dialogHelper.createDialog({
+                        size: 'small',
+                        removeOnClose: true,
+                        scrollY: false
+                    });
+                    dlg.classList.add('ui-body-a');
+                    dlg.classList.add('background-theme-a');
+                    dlg.classList.add('directoryPicker');
+                    dlg.classList.add('formDialog');
+
+                    let html = '';
+                    html += '<div class="formDialogHeader">';
+                    html += '<button is="paper-icon-button-light" class="btnCloseDialog autoSize" tabindex="-1"><span class="material-icons arrow_back"></span></button>';
+                    html += '<h3 class="formDialogHeaderTitle">';
+                    html += options.header || globalize.translate('HeaderSelectPath');
+                    html += '</h3>';
+                    html += '</div>';
+                    html += getEditorHtml(options, systemInfo);
+                    dlg.innerHTML = html;
+                    initEditor(dlg, options, fileOptions);
+                    dlg.addEventListener('close', onDialogClosed);
+                    dialogHelper.open(dlg);
+                    dlg.querySelector('.btnCloseDialog').addEventListener('click', () => {
+                        dialogHelper.close(dlg);
+                    });
+                    this.currentDialog = dlg;
+                    dlg.querySelector('#txtDirectoryPickerPath').value = initialPath;
+                    const txtNetworkPath = dlg.querySelector('#txtNetworkPath');
+                    if (txtNetworkPath) {
+                        txtNetworkPath.value = options.networkSharePath || '';
+                    }
+                    if (!options.pathReadOnly) {
+                        refreshDirectoryBrowser(dlg, initialPath, fileOptions, true);
+                    }
                 }
-            };
-        }
-    }
+            );
+        };
 
-    let systemInfo;
+        close = () => {
+            if (this.currentDialog) {
+                dialogHelper.close(this.currentDialog);
+            }
+        };
+    }
 
 /* eslint-enable indent */
-export default directoryBrowser;
+export {DirectoryBrowser};
```

Comment:
This seems like an unusual way of exporting this class.

Reply:
https://github.com/jellyfin/jellyfin-web/pull/2707/commits/d6c00aa6f40efbf41e0acd6c6398b6f356b787e5